Lacework/lacework-cloud-security/ingest/parser.yml

```
@@ -9,7 +9,13 @@ pipeline:
       properties:
         input_field: "{{json_event.message.alertInfo.description}}"
         output_field: message
-        pattern: " For account: %{DATA:account_name}  : %{GREEDYDATA} %{INT:occurence} times  by user  %{DATA:user_name}    "
+        pattern: "%{HEADER}?\\s*(%{SIGHTING_USER}|%{FIRST_USER}|%{FIRST_SERVICE}|%{AWS_ACCOUNT})\\s*"
```

For some (little) performance improvement, rearrange the regex alternations to move ones with greedydata capture to the end:
```suggestion
        pattern: "%{HEADER}?\\s*(%{FIRST_USER}|%{AWS_ACCOUNT})|%{FIRST_SERVICE}|%{SIGHTING_USER}\\s*"
```